themes/default/content/registry/packages/aws-native/installation-configuration.md

```diff
@@ -135,8 +135,8 @@ As an optional step, if you have multiple AWS profiles set up, you can specify a
 
 ## Configuration options
 
-{{% notes type="warning" %}}
-Right now, some configuration options aren't supported in AWS Native. In the coming months, AWS Native will provide the same set of configuration options as [AWS Classic]({{< relref "/registry/packages/aws" >}}). You can check on the status of a configuration option you need by visiting our backlog of [Github issues](https://github.com/pulumi/pulumi-aws-native/issues?q=is%3Aissue+is%3Aopen+label%3Aprovider-config); feel free to add your vote to an issue by adding a 👍 reaction.
+{{% notes type="info" %}}
+Some configuration options are not currently supported in AWS Native. You can check on the status of a configuration option you need by visiting our backlog of [Github issues](https://github.com/pulumi/pulumi-aws-native/issues?q=is%3Aissue+is%3Aopen+label%3Aprovider-config); feel free to add your vote to an issue by adding a 👍 reaction.
```

Comment:
thx for fixing this up!!

```suggestion
Some configuration options are not supported in AWS Native. You can check on the status of a configuration option you need by visiting our backlog of [Github issues](https://github.com/pulumi/pulumi-aws-native/issues?q=is%3Aissue+is%3Aopen+label%3Aprovider-config); feel free to add your vote to an issue by adding a 👍 reaction.
```

we try and avoid the use of `currently` since it implies it coming, but may never come or not in a reasonable amount of time.

Reply:
I debated using it as I prefer not to have any relative time reference. The items are in the backlog, but they are not prioritized. I will remove it.